BedrockCommand.h

```diff
@@ -146,6 +146,10 @@ class BedrockCommand : public SQLiteCommand {
     // Return the number of commands in existence.
     static size_t getCommandCount() { return _commandCount.load(); }
 
+    // True if this command should be escalated immediately. This can be true for any command that does all of its work
+    // in `process` instead of peek, as it will always be escalated to leader 
+    const bool escalateImmediately;
```

Comment:
should we add protections in peek to make sure this isn't set to true? it could cause errors to set this on a command that has https requests that were never sent, for example.

Reply:
In peek where?

Reply:
heh, good question. talked in slack; worst case failure is really that we don't return early in a follower or an exception occurs in peek on leader instead of a follower.